pkg/apis/configuration/v1alpha1/types.go

```diff
@@ -215,6 +215,9 @@ type MetricSpec struct {
 	HTTP *MetricHTTP `json:"http,omitempty"`
 	// +optional
 	Rules []MetricsRule `json:"rules,omitempty"`
+	// Latency distribution buckets. If not set, the default buckets are used.
+	// +optional
+	LatencyDistributionBuckets []int `json:"latencyDistributionBuckets,omitempty"`
```

Comment:
Please can we add a more detailed specific comment about what this tunes? I can probably _guess_ what this does, but it would be good to include what latency in this context means, what metric types this effects (with some examples), and some example values a user would typically include.

Without reading further, latency of daprd to app is going to be vastly different to daprd to an external flaky HTTP IoT device so a blanket bucket set could be confusing.

Reply:
sure, this is per config, so people can adjust that to the respective app. will provide some more context